source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRActionSetRenderer.mm

```diff
@@ -60,32 +68,40 @@ - (UIView *)renderButtons:(ACRView *)rootView
     ACRRegistration *reg = [ACRRegistration getInstance];
     ACOFeatureRegistration *featureReg = [ACOFeatureRegistration getInstance];
 
-    UIView<ACRIContentHoldingView> *childview = nil;
+    UIStackView *childview = [[UIStackView alloc] init];
```

Comment:
why was the previous design using some of our custom views? what advantages does changing to a UIStackview has?

Reply:
we don't need custom view's extra feature. stock UIStackView is sufficient to get the job done.

---
In reply to: [387950942](https://github.com/microsoft/AdaptiveCards/pull/3818#discussion_r387950942) [](ancestors = 387950942)